src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -700,11 +703,22 @@ private static int compareImports(String import1, String import2) {
      *        Line number of current import.
      * @return true, if token have empty line before.
      */
-    private boolean hasEmptyLineBefore(int lineNo) {
+    private int emptyLinesBefore(int lineNo) {
```

Comment:
The method returns a value. Its name should start with 'get' prefix.
The algorithm requires exact one line before import statement. Why can't we name the methods as 'hasOneLineBefore'?